### ISSUE TITLE
Praise for the goals of SC 1.4.11 Non-text Contrast

### ISSUE THREAD

**Issue by peterkorn** [w3c/wcag21#758]
I share the concerns on evaluation challenges of this SC.  We have been applying a 3:1 contrast ratio to "distinguishing parts" of graphics and found it very helpful and appreciated by customers.  It is worth working very hard to find sufficient techniques and other ancillary guidance to retain this.  Shifting to AAA is also worth considering if you lack confidence in the ability to evaluate compliance, given that most organizations measure themselves on AA but read through to AAA and so will be encouraged to "do right" by this SC without being held to it if evaluation remains ambiguous.

**Comment by awkawk** [comment]
Thank you for commenting. For more information about how the AG WG will process this issue, see the following:
- [Issue Processing](https://www.w3.org/WAI/WCAG21/comments/#issueprocessing) - How the WG will discuss the issue
- [Issue Resolution](https://www.w3.org/WAI/WCAG21/comments/#issueresolution) - Identifying and responding to the official WG response
- [Issue Notification](https://www.w3.org/WAI/WCAG21/comments/#issuenotification) - How to limit notifications from GitHub

**Comment by DavidMacDonald** [comment]
If we were to move it to AAA, I think the interface components bullet should stay at AA.

**Comment by alastc** [comment]
Hi @peterkorn, (non-official response)

Thank you for the comment, if you have any material or practical advice on producing contrasting graphics that you could share, they would be greatly appreciated!

In the meantime, we will work on providing techniques and guidance on this one.

-Alastair

**Comment by alastc** [comment]
@DavidMacDonald my understanding of the process is that any SC would be in or out, there is inbetween.

**Comment by DavidMacDonald** [comment]
@alastc Yeah, our last survey, I almost put a blocker on this, I put in the comment ... these 2 were forced together for expediency and now they are twins which would both die together even though the complaints are against graphics and not components.

**Comment by alastc** [comment]
Well, having been looking at examples for the understanding & techniques, the graphics one isn't that hard. Most sites either pass immediately, or could pass quickly. It's only when you get to complex content graphics that the testing gets harder.

**Comment by awkawk** [comment]
(proposed)

Thank you for the comment, if you have any material or practical advice on producing contrasting graphics that you could share, they would be greatly appreciated, as would any implementation challenges or examples you encounter.

In the meantime, we will work on providing techniques and guidance on this one.

**Comment by awkawk** [comment]
The WG decided on the above response, so we changed the text in the comment containing the proposed response to read "[Official WG Response]". Please confirm is you are satisfied with the response within 1 week. If we haven't heard a response in a week we will regard the resolution as satisfactory.
